docs/snippets.py

```diff
@@ -301,13 +299,13 @@ def main():
     client = storage.Client()
     for example in _find_examples():
         to_delete = []
-        print("%-25s: %s" % _name_and_doc(example))
+        print(f"%-25s: {_name_and_doc(example)}")
```

Comment:
The conversion here doesn't look quite right.
`%-25s` adds 25 space padding to the first item from the tuple. 
Suggested alternative:
```
name, doc: _name_and_doc(example)
print(f"{name:>25}: {doc}"
```